src/zenml/models/model_models.py

```diff
@@ -484,12 +483,16 @@ class ModelVersionArtifactRequestModel(
 ):
     """Model version link with artifact request model."""
 
+    artifact: UUID
+
 
 class ModelVersionArtifactResponseModel(
     ModelVersionArtifactBaseModel, WorkspaceScopedResponseModel
 ):
     """Model version link with artifact response model."""
 
+    artifact: ArtifactResponse
```

Comment:
Seems that this model still lacks the update from the link, also the full artifact response might be a big thing, not worth pulling here. I would seek advice from @bcdurak here, but I would, maybe, limit to only needed fields instead (name, type, user details).
@Cahllagerfeld , what fits better or full response is fine here?

P.S. same applies to the `ModelVersionPipelineRunResponseModel`

Reply:
Full artifact only has `name`, `version`, `type`, and `uri`. I.e., only `uri` is redundant, and IMO this is much cleaner than adding `artifact_name`, `artifact_type` as separate fields into the link.

Reply:
User of the artifact will not be in the response, but the link itself already has a user (which will be the same in 99% of cases) so we can simply display that instead.

Reply:
I don't think it makes sense for @Cahllagerfeld to work on anything related to models before #2066

We already discussed that we'll build him a Docker image including all changes once this PR and #1999 are merged into #2066